content/blog/_posts/inclusive-infra.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Towards More Inclusive Infrastructure"
+date: 2021-04-14
+slug: inclusive-infra
+authors: Reinhardt Quelle
+---
+Posted by Reinhardt Quelle, Dir. Site Reliability Engineering for Cisco Emerging Technolgies and Incubation
+
+This post will be short, because it really is _that simple_.
```

Comment:
```suggestion
```

I get what you're trying to drive at by putting this line front and center, but "simple" is actually a pretty loaded term – what's simple for me might be complex and difficult for someone else to undertake.

Reply:
We had a fair bit of discussion on this, and the emphasis was intentional.  Inclusion is complex. Infrastructure management is complex.  Conversations are difficult.  But, given the complicated things we already undertake - IaaS, linting, common pipelines, and so on, making incremental progress is not just possible, but... simple.

Later, I note "there is much left to do", and there is,  but the first step, as outlined below, is straightforward when we leverage the tools we have.